.vib/harbor-registryctl/goss/harbor-registryctl.yaml

```diff
@@ -0,0 +1,38 @@
+group:
+  harbor:
+    exists: true
+user:
+  harbor:
+    exists: true
+file:
+  /var/lib/registry:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+  /storage:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+command:
+  # Ensure swagger binary doesn't exist
+  check-swagger-binary:
+    exec: ls /opt/bitnami/harbor-registryctl/bin/swagger 2>/dev/null
+    exit-status: 2
+  # Ensure a set of directories exist and the non-root user has write privileges to them
+  check-directories-exist-with-user:
+    exec: ls -dl /etc/ssl/certs /etc/pki/tls/certs/ 2>/dev/null | grep "drwxrwxr-x.*harbor"
+    exit-status: 0
+  # Ensure permissions for Internal TLS
+  check-permissions-system-certs:
+    exec: ls -l /etc/pki/tls/certs/ca-bundle.crt /etc/pki/tls/certs/ca-bundle.trust.crt /etc/ssl/certs/ca-certificates.crt 2>/dev/null | grep ".*-rw-rw-r--.*harbor"
+    exit-status: 0
+  check-harbor-registryctl-binary:
+    exec: harbor_registryctl --help
+    exit-status: 0
+  check-app-version:
+    exec: registry --version | grep -o "[0-9]*\.[0-9]*\.[0-9]"
+    exit-status: 0
+    stdout:
+      - {{ .Env.APP_VERSION }}
```

Comment:
Why not using `common/goss/templates/check-app-version.yaml`? What is the grep needed for here?

```
I have no name!@2ff066db6815:/vib$ registry --version
registry github.com/docker/distribution v2.8.0
```